qcdb/intf_nwchem/harvester.py

```diff
@@ -415,15 +415,12 @@ def format_modelchem_for_nwchem(name, dertype, ropts, sysinfo, verbose=1):
     #          'properties': 'prop',
     #         }[driver]
 
-    runtyp = {0: 'energy',
+    runtyp = {0: 'energy' or 'property',
```

Comment:
Yes, I noticed it always picked up energy, would piping it like `'energy' | 'property'` like in https://github.com/vivacebelles/qcdb/blob/nwc/qcdb/util/regex.py#L33 be fine?

Reply:
No, consider that you're defining the dictionary at runtime. because of the `or`, it's evaluating `'energy'` to T/F (T, since it's not the empty string) and thus never even considers `'property'` (b/c `or`, not `and`). moreover, a dict can't have a single key (0) that's associated with multiple values.

probably the best thing to do is pass in the `model.driver` (energy/gradient/property) rather than `model.driver.derivative_int()` (0/1/2), then you can use it directly.

the pipe in the link is regex syntax.